runtime/shibuya/src/lib.rs

```diff
@@ -1298,7 +1298,10 @@ pub type Executive = frame_executive::Executive<
 /// All migrations that will run on the next runtime upgrade.
 ///
 /// Once done, migrations should be removed from the tuple.
-pub type Migrations = ();
+pub type Migrations = (
+    pallet_preimage::migration::v1::Migration<Runtime>,
```

Comment:
@Dinonard Shibuya pallet_preimage version is 1 but on-chain version is 0. Did we forget to run migration?

Reply:
Quite possibly, I believe it was added during some attempt at a governance experiment.